source/dagger/gameplay/common/simple_collisions.h

```diff
@@ -12,6 +12,9 @@ struct SimpleCollision
     bool is_collidable;
     bool is_moveable;
 
+    float side_x;
+    float side_y;
```

Comment:
Even when adding new things to an already started class, let's not introduce more errata -- I'd be happy if we changed all the types to our own aliased types, so Bool, Float32, etc. instead of pushing native types further.